app/assets/stylesheets/application.scss

```diff
@@ -0,0 +1,197 @@
+/*
+ * This is a manifest file that'll be compiled into application.css, which will include all the files
+ * listed below.
+ *
+ * Any CSS and SCSS file within this directory, lib/assets/stylesheets, vendor/assets/stylesheets,
+ * or any plugin's vendor/assets/stylesheets directory can be referenced here using a relative path.
+ *
+ * You're free to add application-wide styles to this file and they'll appear at the bottom of the
+ * compiled file so the styles you add here take precedence over styles defined in any styles
+ * defined in the other CSS/SCSS files in this directory. It is generally better to create a new
+ * file per style scope.
+ *
+ *= require_tree .
+ *= require_self
+ */
+
+@import "bootstrap-sprockets";
+@import "bootstrap";
+@import "variables";
+
+
+body{
+  font-family: 'Josefin Sans', sans-serif;
+  background-color: $background;
+  margin: 0px;
+  color: $blackish;
+  overflow: scroll;
+}
+
+.content-container{
+  box-sizing: border-box;
+  background: white;
+  color: $blackish;
+  padding: 20px;
+  margin: 15px;
+}
+
+nav{
+  height: 90px;
+  display: block;
+  margin: 0px;
```

Comment:
is this inside content-container?  if not, you don't need to set margin 0 here since body has it